docs/sandbox/VehicleRentalServiceDirectory.md

```diff
@@ -71,6 +71,28 @@ networks are dropped. Note! The values in the "default-network" are not used to
 missing field values in networks listed.
 
 
+<h4 id="vehicleRentalServiceDirectory_networks_0_allowKeepingVehicleAtDestination">allowKeepingVehicleAtDestination</h4>
+
+**Since version:** `2.5` ∙ **Type:** `boolean` ∙ **Cardinality:** `Optional` ∙ **Default value:** `false`   
+**Path:** /vehicleRentalServiceDirectory/networks/[0] 
+
+Enables `allowKeepingVehicleAtDestination` for the given network.
+
+Configures if a vehicle rented from a station must be returned to another one or can
+be kept at the end of the trip.
+
+See the regular [GBFS documentation](../UpdaterConfig.md#GBFS vehicle rental systems) for more information.
```

Comment:
Do these links work in docs.opentripplanner.org ?

Reply:
Good point. The version without the hyphens only works in IntelliJ. When being converted to HTML they need to be hyphenated, which I have done in https://github.com/opentripplanner/OpenTripPlanner/pull/5944/commits/b6d4aeb5689db55d99817986f158ca840e55cac5